ee/hogai/utils/types.py

```diff
@@ -140,3 +140,9 @@ class AssistantNodeName(StrEnum):
     MEMORY_COLLECTOR_TOOLS = "memory_collector_tools"
     INKEEP_DOCS = "inkeep_docs"
     INSIGHT_RAG_CONTEXT = "insight_rag_context"
+    PRODUCT_ANALYTICS_SUBGRAPH = "product_analytics_subgraph"
+
+
+class AssistantMode(StrEnum):
+    ASSISTANT = "assistant"
+    INSIGHTS_TOOL = "insights_tool"
```

Comment:
This is duplicated from `Assistant.Type`, why not use only that one on the model?

Reply:
Didn't understand what you mean by `Assistant.Type`, I can't find that anywhere.

Reply:
Sorry, brain fart when typing, I meant `Conversation.Type`!